src/trade/tradeConf.es6

```diff
@@ -147,26 +137,16 @@ const register_ticks = (state, extra) => {
       }
    }
 
-   let entry = null, expiry = null;
-   let tracking_timeout_set = false;
-   const track_ticks = () => {
-      tracking_timeout_set = false;
-      last_1000_ticks.filter(
-         (tick) => (tick.symbol === extra.symbol && tick.epoch*1 >= entry && tick.epoch*1 <= expiry)
-      ).forEach(add_tick);
-      if(tick_count > 0) {
-         tracking_timeout_set = true;
-         setTimeout(track_ticks, 300);
-      }
-   }
+   let entry = null,
+      expiry = null;
 
    fn = liveapi.events.on('proposal_open_contract', (data) => {
       const contract = data.proposal_open_contract;
       if(contract.contract_id !== extra.contract_id) return;
       entry = contract.entry_tick_time * 1;
       expiry = contract.date_expiry * 1; /* date_expiry gets updated when contract is settled !!! */
-      if(!tracking_timeout_set)
-         track_ticks();
+      if (contract.current_spot_time >= entry)
+         add_tick(data.proposal_open_contract);
```

Comment:
@aminroosta @arnabk I would like to disagree. Please refer to the following screenshot: 

![some](http://i.imgur.com/jxb4VKO.png)

What I'm doing is similar to what was done earlier, i.e; instead of subscribing to the `on('ticks')` event I'am subscribing to the `proposal_open_contract`. In none of the cases we're sending an api request from **tradeConf.es6**, hence making the delay same for both the requests. Also, I would say opting for `proposal_open_contract` is better because on successful buy we first get a _stray tick_ confirming that the contract was bought successfully and then it starts streaming, giving us ample of time to register the `proposal_open_contract` event